test/specs/parser/tests/01.parser.js

```diff
@@ -320,6 +322,22 @@ exports.execute = function(options) {
             });
         });
 
+        describe("when uri have invalid paging Criteria", function() {
+            it("it should throw an error.", function() {
+                expect(function () {
+                    options.ermRest.parse(baseUri + "@after()");
+                }).toThrow(invalidPageCondition);
+            });
+        });
+
+        describe("when uri have invalid paging Criteria", function() {
```

Comment:
This describe doesn't describe the test inside it. This should be something like "when uri has an invalid filter".